### PR TITLE
Add Vietnamese language!

### PR DESCRIPTION
**Pull request by tunglamvghy**

### CODE REVIEW

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,65 @@
+"About HeliPort" = "Thông tin HeliPort";
+"Address: " = "Địa chỉ：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:2]: Please replace Chinese colons with normal colons and a space

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+"Cancel" = "Huỷ bỏ";
+"Check for Updates..." = "Kiểm tra cập nhật...";
+"Create Diagnostics Report..." = "Tạo báo cáo lỗi...";
+"Create Network..." = "Tạo mạng mới...";
+"Disconnect from: " = "Huỷ kết nối：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:7]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+"Dynamic WEP" = "WEP cố định";
+"Enable Wi-Fi Logging" = "Bật ghi lỗi Wifi";
+"Enter the name and security type of the network you want to join." = "Nhập tên và loại bảo mật của mạng mà bạn muốn kết nối";
+"FUNCTION NOT IMPLEMENTED" = "CHỨC NĂNG CHƯA HOẠT ĐỘNG";
+"Find and join a Wi-Fi network." = "Tìm và kết nối một mạng Wi-fi";
+"Interface Name: " = "Tên mạng：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:13]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+"Join Other Network..." = "Kết nối mạng khác...";
+"Join" = "Kết nối";
+"Network Name:" = "Tên mạng：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:16]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+"No Network Avaliable" = "Không có mạng nào hoạt động";
+"No Status Information Avaliable" = "Không có thông tin trạng thái";
+"None" = "Không";
+"Open Network Preferences..." = "Mở Cài đặt mạng...";
+"Open Wireless Diagnostics..." = "Sửa lỗi Wifi...";
+"Password:" = "Mật khẩu：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:22]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+"Password" = "Mật khẩu";
+"Quit HeliPort" = "Thoát";
+"Reachable" = "Có thể truy cập";
+"Remember this network" = "Nhớ mạng này";
+"Security:" = "Bảo mật：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:27]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+"WPA/WPA2 Personal" = "WPA/WPA2 Cá nhân";
+"WPA2 Enterprise" = "WPA2 Doanh nghiệp";
+"WPA2 Personal" = "WPA2 Cá nhân";
+"WPA2/WPA3 Enterprise" = "WPA2/WPA3 Doanh nghiệp";
+"WPA2/WPA3 Personal" = "WPA2/WPA3 Cá nhân";
+"WPA3 Enterprise" = "WPA3 Doanh nghiệp";
+"WPA3 Personal" = "WPA3 Cá nhân";
+"Wi-Fi: Connected" = "Wi-Fi：Đã kết nối";
+"Wi-Fi: Connecting" = "Wi-Fi：Đang kết nối";
+"Wi-Fi: Looking for Networks..." = "Wi-Fi：Đang tìm mạng...";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:47]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+"    Security: "        = "    Bảo mật：";
+"    BSSID: "           = "    BSSID：";
+"    Channel: "         = "    Kênh：";
+"    Country Code: "    = "    Mã quốc gia：";
+"    RSSI: "            = "    RSSI：";
+"    Noise: "           = "    Noise：";
+"    Tx Rate: "         = "    Tỷ lệ Tx：";
+"    PHY Mode: "        = "    Dạng PHY：";
+"    MCS Index: "       = "    Chỉ số MCS：";
+"    NSS: "             = "    NSS：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:65]: Same as above

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+"Remember this network" = "Nhớ mạng này";
+"Security:" = "Bảo mật：";
+"Show password" = "Hiện mật khẩu";
+"Turn Wi-Fi Off" = "Tắt Wi-Fi";
+"Turn Wi-Fi On" = "Bật Wi-Fi";
+"Unavailable" = "Không có sẵn";
+"Unknown" = "Không rõ";
+"Unreachable" = "Không thể truy cập";
+"Username:" = "Tên người dùng：";
+"Version: " = "Phiên bản：";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:35]: Same as above

**Comment by williambj1** [comment]
CC @tunglamvghy

**Comment by williambj1** [comment]
@tunglamvghy why closed?

File: HeliPort.xcodeproj/project.pbxproj [hunk 10]
```diff
@@ -80,6 +80,7 @@
 		7557CDF324935260000F0F71 /* Credits.rtf */ = {isa = PBXFileReference; lastKnownFileType = text.rtf; path = Credits.rtf; sourceTree = "<group>"; };
 		7562BBE324826102008A9BD2 /* README.md */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
 		75FDF38A2481D22000B2A601 /* NetworkManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = NetworkManager.swift; sourceTree = "<group>"; };
+		A6B1F45424B9C60A00EBBE79 /* vi */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = vi; path = vi.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:83]: Please specify file line ending and encoding
image

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+"    Security: "        = "    Bảo mật: ";
+"    BSSID: "           = "    BSSID: ";
+"    Channel: "         = "    Kênh: ";
+"    Country Code: "    = "    Mã quốc gia: ";
+"    RSSI: "            = "    RSSI: ";
+"    Noise: "           = "    Noise: ";
+"    Tx Rate: "         = "    Tỷ lệ Tx: ";
+"    PHY Mode: "        = "    Dạng PHY: ";
+"    MCS Index: "       = "    Chỉ số MCS: ";
+"    NSS: "             = "    NSS: ";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:65]: Please remove extra white spaces

Comment (tunglamvghy) [reply HeliPort/Appearance/vi.lproj/Localizable.strings:65]: Okie. I rarely use Gihub so I dont know how to update this file?

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 12]
```diff
@@ -1,16 +1,16 @@
 "About HeliPort" = "Thông tin HeliPort";
-"Address: " = "Địa chỉ: ";
+"Address: " = "Địa chỉ:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 13]
```diff
@@ -1,16 +1,16 @@
...
 "Cancel" = "Huỷ bỏ";
 "Check for Updates..." = "Kiểm tra cập nhật...";
 "Create Diagnostics Report..." = "Tạo báo cáo lỗi...";
 "Create Network..." = "Tạo mạng mới...";
-"Disconnect from: " = "Huỷ kết nối: ";
+"Disconnect from: " = "Huỷ kết nối:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 14]
```diff
@@ -1,16 +1,16 @@
...
 "Dynamic WEP" = "WEP cố định";
 "Enable Wi-Fi Logging" = "Bật ghi lỗi Wifi";
 "Enter the name and security type of the network you want to join." = "Nhập tên và loại bảo mật của mạng mà bạn muốn kết nối.";
 "FUNCTION NOT IMPLEMENTED" = "CHỨC NĂNG CHƯA HOẠT ĐỘNG";
 "Find and join a Wi-Fi network." = "Tìm và kết nối một mạng Wi-fi.";
-"Interface Name: " = "Tên mạng: ";
+"Interface Name: " = "Tên mạng:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 15]
```diff
@@ -19,17 +19,17 @@
 "None" = "Không";
 "Open Network Preferences..." = "Mở Cài đặt mạng...";
 "Open Wireless Diagnostics..." = "Sửa lỗi Wifi...";
-"Password:" = "Mật khẩu: ";
+"Password:" = "Mật khẩu:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 16]
```diff
@@ -19,17 +19,17 @@
...
 "Password" = "Mật khẩu";
 "Quit HeliPort" = "Thoát";
 "Reachable" = "Có thể truy cập";
 "Remember this network" = "Ghi nhớ mạng này";
-"Security:" = "Bảo mật: ";
+"Security:" = "Bảo mật:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 17]
```diff
@@ -19,17 +19,17 @@
...
 "Show password" = "Hiển thị mật khẩu";
 "Turn Wi-Fi Off" = "Bật Wi-Fi";
 "Turn Wi-Fi On" = "Tắt Wi-Fi";
 "Unavailable" = "Không có sẵn";
-"Unknown" = "Không rõ";
+"Unknown" = "Không rõ";s
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: `s`?

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 18]
```diff
@@ -50,16 +50,16 @@
...
-"    Internet: "        = "    Internet: ";
-"    Security: "        = "    Bảo mật: ";
-"    BSSID: "           = "    BSSID: ";
-"    Channel: "         = "    Kênh: ";
-"    Country Code: "    = "    Mã quốc gia: ";
-"    RSSI: "            = "    RSSI: ";
-"    Noise: "           = "    Noise: ";
-"    Tx Rate: "         = "    Tỷ lệ Tx: ";
-"    PHY Mode: "        = "    Dạng PHY: ";
-"    MCS Index: "       = "    Chỉ số MCS: ";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings]: Please don't remove necessary spaces after the colons

File: HeliPort/Appearance/vi.lproj/Localizable.strings [hunk 19]
```diff
@@ -50,16 +50,16 @@
...
+"    Security: "        = "    Bảo mật:";
+"    BSSID: "           = "    BSSID:";
+"    Channel: "         = "    Kênh:";
+"    Country Code: "    = "    Mã quốc gia:";
+"    RSSI: "            = "    RSSI:";
+"    Noise: "           = "    Noise:";
+"    Tx Rate: "         = "    Tỷ lệ Tx:";
+"    PHY Mode: "        = "    Dạng PHY:";
+"    MCS Index: "       = "    Chỉ số MCS:";
+"    NSS: "             = "    NSS:";
```
Comment (williambj1) [inline HeliPort/Appearance/vi.lproj/Localizable.strings:65]: Please don't remove necessary spaces after the colons

Comment (tunglamvghy) [reply HeliPort/Appearance/vi.lproj/Localizable.strings]: I've just edited it.

**Pull request merged**